src/openms/include/OpenMS/FORMAT/MzMLFile.h

```diff
@@ -175,16 +175,25 @@ namespace OpenMS
     */
     bool isSemanticallyValid(const String& filename, StringList& errors, StringList& warnings);
 
+
+    struct SpecInfo
```

Comment:
better name? seems to be tied to getCentroidInfo

Reply:
put Tandem somewhere? TandemSpecCounts ?

Reply:
this is perfectly applicable to MS1 as well...